### ISSUE TITLE
Scope and future of the OSLC Browser?

### ISSUE THREAD

**Issue by berezovskyi** [OSLC/oslc-browser#19]
After managing to get OSLC Browser to work on OSLC CM RefImpl server (via https://github.com/OSLC/oslc-browser/pull/18; https://github.com/oslc-op/refimpl/commit/66527f65a3f16e7785a303ecb1bf4d179d6d9feb; https://github.com/oslc-op/website-content-negotiation/issues/8), I have gotten this output:

<img width="1200" alt="Screenshot 2020-12-11 at 22 40 06" src="https://user-images.githubusercontent.com/64734/101965632-b266e680-3c15-11eb-85e8-2124705a3f8e.png">

I am trying to understand now how this browser is to be user and what is its utility? It does not seem to handle RootServices, and does not list the SPC properly. I got a bit further with the linked resources from RefImpl:

<img width="1050" alt="Screenshot 2020-12-12 at 01 15 38" src="https://user-images.githubusercontent.com/64734/101966320-9d8b5280-3c17-11eb-9f4f-1e665405885c.png">

Questions:

1. Is that it? Or am I holding it wrong™?
2. Do we have any interest in improving it? I was thinking to at least upgrade some dependencies and then jumped off my seat when I saw #15.
3. @jadelkhoury and @axelreichwein what do you think about OSLC in the browser in general? You had first-hand experience with https://github.com/kth-mda/oslc-schema-viewer (+https://github.com/kth-mda/slviewer and https://github.com/kth-mda/rdfexplorer) and https://github.com/koneksys/KLD. I personally see 3 promising things: 
	1. Solid is pushing RDF/LDP on the web and they are not suffering from our CORS/CSP misery.
	1. WebComponents spec is essenntialy IFRAME 2.0 for 2020 web and given OSLC's most popular feature is DUI, we are ought to look into it.
	1. I hear about more and more modelling tools moving to the web itself, including https://github.com/eclipse-sirius/sirius-web

**Comment by jamsden** [comment]
You should be seeing a list of resources, maybe you picked a bad example. Try selecting a requirement, see its implemented by change requests, then its validated by test cases. When you see a resource, you should be able to expand the accordion to see all its links, then click on the link to see the list of related resources in the next column to the right. 

Clicking on the resource will show it in the graph view where its related resources can be shown and followed.

I picked React and Carbon because that's what is being adopted by IBM. 

I have never see a web modeling application I would want to use for more than 2 min. There's nothing wrong with desktop applications.

**Comment by jamsden** [comment]
The utility of the browser is that it can navigate any OSLC resource, following any of its links to any other resources, and is independent of what server is providing that resource. It allows you to see related elements and quickly follow links to browse related resources in some system.

**Comment by berezovskyi** [comment]
> The utility of the browser is that it can navigate any OSLC resource, following any of its links to any other resources, and is independent of what server is providing that resource. It allows you to see related elements and quickly follow links to browse related resources in some system.

Right, so it's not designed for SPC/rootservices discovery then. Got it.

> You should be seeing a list of resources, maybe you picked a bad example. Try selecting a requirement, see its implemented by change requests, then its validated by test cases. When you see a resource, you should be able to expand the accordion to see all its links, then click on the link to see the list of related resources in the next column to the right.

I see, I just don't have enough links then. 

> Clicking on the resource will show it in the graph view where its related resources can be shown and followed.

I only got it to show a single node with the resource itself. 

> I have never see a web modeling application I would want to use for more than 2 min. There's nothing wrong with desktop applications.

Same here actually except for draw.io (I used OmniGraffle before, which is arguable the best app you can get on macOS for diagramming) but it's not a modelling tool to be honest. With LyoD, I would like to see something that runs from command line more than a web ui.

**Comment by berezovskyi** [comment]
I put some updates/remarks in my top comment under the Questions section.
